rktl_planner/launch/simple_agent.launch

```diff
@@ -0,0 +1,19 @@
+<launch>
+    <arg name="agent_name" default="agent0"/>
+    <arg name="car_name" default="car0"/>
+
+    <group ns="agents/$(arg agent_name)">
```

Comment:
I think we should just put the autonomy stuff in the same namespace as the cars. Would make things much simpler for topics

Reply:
The plan was to separate `autonomy` to avoid ruling out a multi-car agent. It's also nice to separate, because all `car` topics and nodes can be placed in a single launch file under `rktl_control` while `autonomy` or `agent` topics are all bundled in their own launch file.

Reply:
I think it is nice because rather than doing:
```
        self.bot_velocity_cmd = rospy.Publisher(f'/cars/{car_name}/command',
            ControlCommand, queue_size=1)

        # Subscribers
        rospy.Subscriber(f'/cars/{car_name}/odom', Odometry, self.odom_cb)
```

We can just directly subscribe to `command` and `odom` in the current namespace. I'm a big fan of not hardcoding anything in the Python source, and using remaps / namespacing in the launch files.

However, if you feel strongly about this, I don't think the existing system is wrong. I just think the other way would be cleaner.

Reply:
At the same time, there are so many nodes and things going on in there, maybe we just leave it as is since it nicely spaces things. I didn't realize it was more than 2 nodes and topics. For the neural network one, I'd like to put it in car since it is just one node.